sql/_27_banana_qa/issue_5765_timezone_support/_01_table_operations/_02_alter/answers/_01_alter_add_hard_default_yes_dttz.answer

```diff
@@ -131,8 +131,8 @@ a    c_dttz1    c_dttz2    c_dttz3
 0
 ===================================================
```

Comment:
Shouldn't it be changed to the same content below?
```
sql/_27_banana_qa/issue_5765_timezone_support/_01_table_operations/_02_alter/answers/_01_alter_add_hard_default_yes_dttz.answer_win
```

Reply:
Reflected the review. Please check again.